tests/wdio.conf.js

```diff
@@ -40,7 +40,8 @@ const baseConfig = {
   // will be called from there.
   //
   specs: [
-    './tests/e2e/**/*.wdio-spec.js'
+    // './tests/e2e/**/*.wdio-spec.js',
```

Comment:
Remember to revert this one when you've finish testing :)

Reply:
Oops, thanks!